api/v1alpha1/limitador_types.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2020 Red Hat.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// LimitadorSpec defines the desired state of Limitador
+type LimitadorSpec struct {
+	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// +optional
+	Replicas int `json:"replicas,omitempty"`
+
+	// +optional
+	Version string `json:"version,omitempty"`
```

Comment:
Just a thought for the long-term (not needed to be tackled for now of course):

Do you intend to provide support for upgrades between versions of Limitador via the operator? If so, letting freely specify the version might imply some challenges in that regard (vs having a hardcoded one)

Reply:
That's a good point. To be honest, I haven't thought about versions, upgrades, etc yet. For now, I'll keep the version field to allow users to install specific versions and then we'll see.

Reply:
:+1: